temba/externals/views.py

```diff
@@ -77,3 +79,23 @@ class Delete(DependencyDeleteModal):
         cancel_url = "@orgs.org_home"
         success_url = "@orgs.org_home"
         success_message = _("Your external service has been deleted.")
+
+
+class GetExternalTypes(APIView):
+    authentication_classes = []
+    permission_classes = []
+    pagination_class = None
+    renderer_classes = [JSONRenderer]
+    throttle_classes = []
+
+    @action(methods=["get"], detail=False)
```

Comment:
Is this action really necessary? Here's an example of how an APIview works: https://www.django-rest-framework.org/api-guide/views/